From ebe5eeca57a9202aba5e6cb2f6984366af011633 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Fri, 3 Aug 2018 20:01:36 +0800
Subject: [PATCH 197/205] arm: dts: am335x: disabled wlcore since wl hardware
 updates

Since wl127x has been dropped in the latest SDK, so do we.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/am335x-evm.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/am335x-evm.dts b/arch/arm/boot/dts/am335x-evm.dts
index e5cb8ec..66aab24 100644
--- a/arch/arm/boot/dts/am335x-evm.dts
+++ b/arch/arm/boot/dts/am335x-evm.dts
@@ -756,7 +756,7 @@
 	dmas = <&edma_xbar 12 0 1
 		&edma_xbar 13 0 2>;
 	dma-names = "tx", "rx";
-	status = "okay";
+	status = "disabled";
 	vmmc-supply = <&wlan_en_reg>;
 	bus-width = <4>;
 	pinctrl-names = "default";
-- 
1.7.9.5

